third_party/move/move-compiler-v2/src/pipeline/explicit_drop.rs

```diff
@@ -79,8 +101,11 @@ impl<'a> ExplicitDropTransformer<'a> {
     /// Add explicit drops at the given code offset.
     fn explicit_drops_at(&mut self, code_offset: CodeOffset, bytecode: &Bytecode) {
         match bytecode {
```

Comment:
Add a comment here why we don't add explicit drops at the given code offset for these bytecode variants, or update the doc string for this function.

Reply:
done